storage/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogManagerConfig.java

```diff
@@ -56,28 +56,28 @@ public final class RemoteLogManagerConfig {
 
 
     public static final String REMOTE_LOG_STORAGE_SYSTEM_ENABLE_PROP = "remote.log.storage.system.enable";
-    public static final String REMOTE_LOG_STORAGE_SYSTEM_ENABLE_DOC = "Whether to enable tier storage functionality in a broker or not. Valid values " +
-            "are `true` or `false` and the default value is false. When it is true broker starts all the services required for tiered storage functionality.";
+    public static final String REMOTE_LOG_STORAGE_SYSTEM_ENABLE_DOC = "Whether to enable remote storage functionality in a broker or not. Valid values " +
```

Comment:
Let us keep this as "tiered storage"

Reply:
Hi @satishd 
It is on my [suggestion](https://github.com/apache/kafka/pull/14266#issuecomment-1687769802) that @tinaselenge made this change. I suggested that because all our publicly facing configurations are named as "remote storage" and same for code related files such as RemoteLogManager. Is there a reason we want to use Tiered storage nomenclature?

I don't have a strong opinion on either but want to ensure that we stick to one nomenclature in the code base and public docs.

Reply:
Thanks @divijvaidya for the clarification. Let us continue having "tiered storage" as mentioned earlier. cc @tinaselenge